eap73-rhel8-byos-vmss/scripts/jbosseap-setup-redhat.sh

```diff
@@ -144,12 +144,11 @@ echo 'WILDFLY_SERVER_CONFIG=standalone-azure-ha.xml' >> $EAP_RPM_CONF_STANDALONE
 echo "Setting configurations in $EAP_LAUNCH_CONFIG"
 echo -e '\t-> JAVA_OPTS=$JAVA_OPTS -Djboss.bind.address=0.0.0.0' | log; flag=${PIPESTATUS[0]}
 echo -e '\t-> JAVA_OPTS=$JAVA_OPTS -Djboss.bind.address.management=0.0.0.0' | log; flag=${PIPESTATUS[0]}
-echo -e '\t-> JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address.private=$(hostname -I)"' | log; flag=${PIPESTATUS[0]}
-echo -e '\t-> JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address.private=$(hostname -I)"' | log; flag=${PIPESTATUS[0]}
+echo -e "\t-> JAVA_OPTS=\"\$JAVA_OPTS -Djboss.bind.address.private=$private_ip\"" | log; flag=${PIPESTATUS[0]}
 
 echo -e 'JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address=0.0.0.0"' >> $EAP_LAUNCH_CONFIG | log; flag=${PIPESTATUS[0]}
 echo -e 'JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address.management=0.0.0.0"' >> $EAP_LAUNCH_CONFIG | log; flag=${PIPESTATUS[0]}
-echo -e 'JAVA_OPTS="$JAVA_OPTS -Djboss.bind.address.private=$(hostname -I)"' >> $EAP_LAUNCH_CONFIG | log; flag=${PIPESTATUS[0]}
+echo -e "JAVA_OPTS=\"\$JAVA_OPTS -Djboss.bind.address.private=$private_ip\"" >> $EAP_LAUNCH_CONFIG | log; flag=${PIPESTATUS[0]}
```

Comment:
If a user decides to create an image from existing VM it will be assigned the original private address (which is wrong). It needs to be resolved every time.

Moreover it blocks our testing. We create image from the template deployment and our testsuite consumes it.